Makefile

```diff
@@ -30,6 +30,16 @@ golangci-lint: mise yq ## Download golangci-lint locally if necessary.
 	@$(MISE) plugin install --yes -q golangci-lint
 	@$(MISE) install -q golangci-lint@$(GOLANGCI_LINT_VERSION)
 
+# ------------------------------------------------------------------------------
+# CI
+# ------------------------------------------------------------------------------
+
+
+.PHONY: kong.supported-versions
+kong.supported-versions:
+	@curl -s https://docs.konghq.com/_api/gateway-versions.json | \
+		jq '[.[] | select(.endOfLifeDate > (now | strftime("%Y-%m-%d")))] | [.[].tag]'
```

Comment:
An unreleased version could be returned from this endpoint. E.g. now `3.10` is not returned from this `jq` query because it has no `endOfLifeDate`, but I'm unsure if this will always be true. Maybe we could explicitly filter out entries with `label: unreleased`?

Reply:
Good catch. 36fc97cd83248274c5ef8f7f8d4617e9314d5078 should handle that.